ui/app/AppLayouts/Communities/controls/Options.qml

```diff
@@ -49,33 +49,51 @@ Column {
 
         StatusBaseText {
             Layout.fillWidth: true
-            text: qsTr("Request to join required")
+            text: qsTr("Any member can pin a message")
             font.pixelSize: Theme.primaryTextFontSize
             TapHandler {
-                onTapped: requestToJoinToggle.toggle()
+                onTapped: pinMessagesToggle.toggle()
             }
         }
 
         StatusCheckBox {
-            id: requestToJoinToggle
+            id: pinMessagesToggle
         }
     }
 
-    RowLayout {
+    ColumnLayout {
         width: parent.width
-        height: d.optionHeight
+        height: d.optionHeight + (requestToJoinToggle.checked ? (warningText.height) : 0)
```

Comment:
This shouldn't be necessary; the height will adjust based on what elements are visible

Reply:
Really needed?